docs/template-windows.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: windows-vm-template
+  annotations:
+    description: "OCP kubevirt windows VM template"
+    tags: "kubevirt,ocp,template,windows"
+objects:
+- apiVersion: kubevirt.io/v1alpha1
+  kind: OfflineVirtualMachine
+  metadata:
+    name: windows-vm-${NAME}
+  spec:
+    template:
+      metadata:
+      labels:
+      spec:
+        terminationGracePeriodSeconds: 0
+        domain:
+          features:
+            acpi: {}
+            apic: {}
+            hyperv:
+              relaxed: {}
+              vapic: {}
+              spinlocks:
+                spinlocks: 8191
+          clock:
+            utc: {}
+            timer:
+              hpet:
+                present: false
+              pit:
+                tickPolicy: delay
+              rtc:
+                tickPolicy: catchup
+              hyperv: {}
+          firmware:
+            uid: 5d307ca9-b3ef-428c-8861-06e72d69f223
```

Comment:
Tricky. This would lead to all windows vms to have the same firwmare uuid.

I wonder if we could have a variable as well here. Maybe CFME has a uuid field? @jhernand

Reply:
right, good catch

Reply:
ManageIQ has an internal identifier for the virtual machine. But it isn't an UUID, and it is assigned *after* the `OfflineVirtualMachine` is created: ManageIQ receives the notification from Kubernetes and then creates its own VM object.

If really required, then ManageIQ could generate an UUID when creating the `OfflineVirtualMachine`, but that is the kind of detail that the Kubevirt client should not be aware of. Can't we just remove this attribute from the template? Will Kubevirt assign then a new UUID to the `OffileVirtualMachine`?

Reply:
Yes, we can do that.

We can remove it from the template, and have an initializer who is adding it to the OfflineVM upon creation if not specified.

@rmohr does this sound ok?

Reply:
/cc @petrkotas